sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/parser/BeamSqlParserTest.java

```diff
@@ -63,7 +64,7 @@ public void testParseCreateTable_withoutType() throws Exception {
     parseTable(
         "create table person (\n"
             + "id int COMMENT 'id', \n"
-            + "name varchar(31) COMMENT 'name') \n"
+            + "name varchar COMMENT 'name') \n"
```

Comment:
<!--new_thread; commit:4a38a26e2bfa0631ef5426455a57802eb81e5e93; resolved:0-->
OK, I commented on the CLI test, but this is the better place for the parser tests.

Reply:
Adding QuickCheck tests